plenum/server/replica.py

```diff
@@ -1034,7 +1042,7 @@ def addToPrePrepares(self, pp: PrePrepare) -> None:
         self.stats.inc(TPCStat.PrePrepareRcvd)
         self.tryPrepare(pp)
 
-    def hasPrepared(self, request) -> bool:
+    def has_pre_prepared(self, request) -> bool:
```

Comment:
But the method does not what it says. It checks whether we've already sent Prepare. According to what it does, the method should be called `has_sent_prepare`